src/shelly_hap_rgb.cpp

```diff
@@ -212,38 +211,84 @@ void RGBWLight::HSVtoRGBW(const HSV &hsv, RGBW &rgbw) const {
   }
 }
 
-void RGBWLight::SetOutputState(const char *source) {
-  LOG(LL_INFO,
-      ("state: %s, brightness: %i, hue: %i, saturation: %i", OnOff(cfg_->state),
-       cfg_->brightness, cfg_->hue, cfg_->saturation));
+void RGBWLight::TurnOn(const std::string &source) {
```

Comment:
This explicit function feels over engineered, my not simply using `UpdateOnOff(true, source)` direct?
Same for `TurnOff` and `Toogle`